packages/plugins/users-permissions/admin/src/pages/Roles/ListPage/index.js

```diff
@@ -55,6 +91,64 @@ const RoleListPage = () => {
             </CheckPermissions>
           }
         />
+        {isLoading ? (
```

Comment:
Looks like you're missing the empty state 😊 . What does it look like when there's no result in the table? I suggest to rely on an empty state for this, can you try to do something like 

```js
<EmptyStateLayout icon={<Loader />} content="Loading your roles..." />
```

and check how it looks like?

Reply:
You might need to use the CustomContentLayout from the helper plugin as well to change the no permissions state.